src/blast_website/BlastUtils.ts

```diff
@@ -0,0 +1,38 @@
+const params: { [key: string]: string } = {
+  dataset: 'uniprotkb_refprotswissprot',
+  threshold: '10',
+  matrix: '',
+  blastFilter: 'false',
+  gapped: 'true',
+  alignments: '250',
+  redirect: 'yes',
+  landingPage: 'no',
+  url2: '',
+};
+
+const getHiddenField = (key: string, value: string) => {
```

Comment:
I think `createHiddenField` is more correct for this function.

Reply:
Renamed!